### DISCUSSION TITLE
Imaginary Container Stuck in "Starting" Status - Imaginary unresponse

### DISCUSSION THREAD

**Discussion by jakepi84** [nextcloud/all-in-one#2261] [Questions] [answered]
I am on Nextcloud AIO v4.7.0. This issue has started in the past week and I am really at a loss. According to the AIO Admin Interface Imginary is "starting" and stuck starting indefinitely. I've tried stopping all containers and restarting, I've deleted the imaginary container and repulled, nothing seems to get it's status to change.

My Nextcloud Log is filled with " cURL error 7: Failed to connect to nextcloud-aio-imaginary port 9000 after 5 ms: Connection refused (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for ..."

The logs for the imaginary container are empty, I only have one line in portainer: "No log line matching the '' filter"

If I access the console for the nextcloud container I am able to ping the imaginary container, but running a curl also yields a connection refused. So it does not appear to be connectivity related.

_bash-5.1# ping nextcloud-aio-imaginary
PING nextcloud-aio-imaginary (172.26.0.7): 56 data bytes
64 bytes from 172.26.0.7: seq=0 ttl=64 time=0.332 ms
64 bytes from 172.26.0.7: seq=1 ttl=64 time=0.232 ms
^C
--- nextcloud-aio-imaginary ping statistics ---
2 packets transmitted, 2 packets received, 0% packet loss
round-trip min/avg/max = 0.232/0.282/0.332 ms
bash-5.1# curl http://nextcloud-aio-imaginary:9000/
curl: (7) Failed to connect to nextcloud-aio-imaginary port 9000 after 1 ms: Connection refused_

I poked around in the imaginary container, but I can't seem to find any logs for imaginary that would help me troubleshoot. 

I do see that imaginary is running in the container at least, but not sure why it seems to be unresponsive.

_nobody@bc85a252b872:/$ ps -aux | grep imag
nobody         1  0.0  0.1 942856 58324 ?        Ssl  06:54   0:00 /usr/local/bin/imaginary -return-size -max-allowed-resolution 222.2_

Not sure where to start looking? Thank you!

**Comment by szaimen** [comment, marked as answer]
Hi, I fear I cannot reproduce this. For me it looks like this:
```
bash-5.1# curl http://nextcloud-aio-imaginary:9000/
{"imaginary":"dev","bimg":"1.1.7","libvips":"8.12.2"}bash-5.1#
```
and then I have this in the imaginary logs:
```
172.18.0.8 - - [03/Apr/2023 11:52:21] "GET / HTTP/1.1" 200 53 0.0010
```

**Reply by jakepi84** [reply]
Not sure what happened, but this morning I rebooted the entire host and it is working again. Thank you for the assistance!

**Reply by szaimen** [reply]
Great that it works now!
